pkg/cmd/pr/shared/finder.go

```diff
@@ -57,7 +58,10 @@ func NewFinder(factory *cmdutil.Factory) PRFinder {
 		branchFn:   factory.Branch,
 		remotesFn:  factory.Remotes,
 		httpClient: factory.HttpClient,
-		progress:   factory.IOStreams,
+		pushDefault: func() (string, error) {
+			return factory.GitClient.Config(context.Background(), "push.default")
+		},
+		progress: factory.IOStreams,
```

Comment:
It looks like `progress` became unaligned here, causing the diff to be noisier than needed.

Reply:
I'm afraid I didn't quite follow. Could you elaborate a bit? Is it related to my PR?

Reply:
Sorry, I mean the whitespace between `progress:` and `factory` changed, which is why the diff looks this way. Does `go fmt` realign them?

Reply:
Nah it was `go fmt` that did this in the first place, presumably because it was separated from the lines above.